forte/data/data_pack.py

```diff
@@ -47,13 +46,13 @@
 from forte.data.ontology.core import EntryType
 from forte.data.ontology.top import (
     Annotation,
+    Grids,
```

Comment:
`Grids` is still an entry? I thought we discuss that it should be a data structure. Or is that change not merged?

Reply:
It's not merged yet and it's in https://github.com/asyml/forte/pull/827